config/storybook/preview-head.html

```diff
@@ -0,0 +1,6 @@
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+<style>
+html, body {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
I too have a file like this in #137 that attempts to better mimic the app wrappers we have in pretty much every app. Definitely seems appropriate to have one though.